src/service-worker/registration.js

```diff
@@ -1,9 +1,7 @@
-import runtime from 'serviceworker-webpack-plugin/lib/runtime'
```

Comment:
If we remove this, we do not need to use the plugin. But we will need to change the webpack config to add manually the service worker file. You can check the plugin codebase to understand how they do it

Reply:
I've already restored the plugin.